### DISCUSSION TITLE
Having this issue in next.js

### DISCUSSION THREAD

**Discussion by bimantadh** [vercel/next.js#59907] [Help]
### Summary

![asd](https://github.com/vercel/next.js/assets/152242985/7eefc204-fc65-4e38-8078-cefeb468e872)

### Additional information

```js
i have added .babelrc file still same issuue is seen. Please do help
```

**Comment by ananuness** [comment]
Have you tried delete the `package-lock.json` file and the `node_modules` dir in your project and then run `npm i` and `npm run dev`? This did help when this occurred with me. 

Hope it helps, if not, don't hesitate to ask!

**Reply by bimantadh** [reply]
Thank you so much. It did work .Thank you for your time

**Reply by ananuness** [reply]
You're welcome!! I'm glad it worked! 😊
